torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -601,10 +601,14 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 return g.call_function(ttnn.log, (softmax_node,), kwargs)
 
             if node.target == torch.ops.aten.div.Tensor:
-                if not isinstance(args[1], (float, int)) and (get_shape(args[0]) != get_shape(args[1])):
-                    recip = g.call_function(ttnn.reciprocal, (args[1],), {})
-                    return g.call_function(ttnn.mul, (args[0], recip), {})
-                return g.call_function(ttnn.div, args, {})
+                if isinstance(args[1], (float, int)):
+                    return g.call_function(ttnn.mul, (args[0], 1.0 / args[1]), {})
```

Comment:
Done!  I checked the additional cases.  Some constant denominators are `int`.  I handle them here as well.